src/arithmetic/entity_funcs/entity_funcs.c

```diff
@@ -219,14 +218,23 @@ SIValue AR_PROPERTY(SIValue *argv, int argc, void *private_data) {
 		// Retrieve the property.
 		SIValue *value = GraphEntity_GetProperty(graph_entity, prop_idx);
 		return SI_ConstValue(value);
-	} else {
+	} else if(SI_TYPE(obj) & T_MAP) {
 		// retrieve map key
 		SIValue key = argv[1];
 		SIValue value;
 
 		Map_Get(obj, key, &value);
 		// Return a volatile copy of the value, as it may be heap-allocated.
 		return SI_ShareValue(value);
+	} else if(SI_TYPE(obj) & T_POINT) {
+		// retrieve property key 
+		SIValue key = argv[1];
+		SIValue value;
```

Comment:
unused.

Reply:
You are right. Done. [ec24b08](https://github.com/RedisGraph/RedisGraph/pull/2524/commits/ec24b08b987aeb67bd9a176dd8c24c2049a78ce3)